pkg/jx/cmd/get_apps.go

```diff
@@ -117,7 +189,21 @@ func (o *GetAppsOptions) generateTable(apps *v1.AppList, kubeClient kubernetes.I
 func (o *GetAppsOptions) generateTableHeaders(apps *v1.AppList) table.Table {
 	t := o.createTable()
 	t.Out = o.CommonOptions.Out
-	titles := []string{"Name", "Version", "Description", "Chart Repository"}
+	titles := []string{"Name", "Version", "Chart Repository", "Status", "Description"}
```

Comment:
I am still missing the namespace though. I installed the JaCoCo App into two different namespaces and the result of get apps is:
```
$jx get apps
Name                  Version   Chart Repository                Status   Description
jx-app-jacoco         1.0.0-dev http://127.0.0.1:8879           DEPLOYED A Helm chart for the JenkinsX JaCoCo App
jx-app-test-lifecycle 0.0.2     http://chartmuseum.jenkins-x.io DEPLOYED A Helm chart for Kubernetes
jx-app-jacoco         1.0.0-dev http://127.0.0.1:8879           DEPLOYED A Helm chart for the JenkinsX JaCoCo App
```

Now I do a:
```
$jx get apps --status jx-app-jacoco
Provide only one app for status check
```
This is confusing, since it is not obvious to me how to differentiate the two JaCoCo Apps. Effectively, I need for example:
```
$ jx get apps --status jx-app-jacoco -n jx
```

@pmuir it is a valid usecase to install an App into multiple namespaces, right?

Reply:
Ah, I thought we were going to do this in a different issue. 

I'll change it tomorrow if we need to show the ns.

Reply:
Not sure whether one needs to enforce it. It gets cumbersome for the default case where everything is in the default namespace. 

Displaying the namespace will to start with make things transparent and give the user a pointer on to use the namespace flag. I think this is not such a big mental jump.

Apart from this, one could think about more specific messages. Instead of having one generic message if multiple apps match, one could for the case of same app in multiple namespaces respond with: "The specified app is installed in multiple namespaces. Use the namespace flag to further qualify the app" (something like this).

Reply:
Finally, as we talked, the info shown in `--status` will always be the resources of the app when it was first installed, so there's no point in filtering this by namespace. 

Also added a better message when multiple different applications are passed to `--status`.

Reply:
Not filtering, but displaying right? I still can have two different apps installed into different namespaces in which case it would be nice to know where to look.

Reply:
The namespace will still appear in the table when executing `jx get apps` and could be filtered.

This only applies when you add `--status`, as there isn't really much of a point to filter / show the namespace, as  atm it's showing the same for all installed apps.